src/pybind/mgr/cephadm/module.py

```diff
@@ -1925,34 +1925,11 @@ def remove_service(self, service_name: str) -> str:
             if self.spec_store[service_name].spec.service_type in ('mon', 'mgr'):
                 return f'Unable to remove {service_name} service.\n' \
                        f'Note, you might want to mark the {service_name} service as "unmanaged"'
-
-        # Report list of affected OSDs
-        osds_msg = {}
-        if service_name.startswith('osd.'):
-            for h, dm in self.cache.get_daemons_with_volatile_status():
-                osds_to_remove = []
-                for name, dd in dm.items():
-                    if dd.daemon_type == 'osd' and (dd.service_name() == service_name or not dd.osdspec_affinity):
-                        osds_to_remove.append(str(dd.daemon_id))
-                if osds_to_remove:
-                    osds_msg[h] = osds_to_remove
-
-        found = self.spec_store.rm(service_name) or osds_msg
-        if found:
-            self._kick_serve_loop()
-            if osds_msg:
-                return f'The service {service_name} will be deleted once the following OSDs: {osds_msg} ' \
-                       f'will be deleted manually.'
-            else:
-                return f'Removed service {service_name}'
-        else:
-            # must be idempotent: still a success.
-            try:
-                self.cache.get_daemon(service_name)
-                return (f'Failed to remove service <{service_name}>. "{service_name}" is the name of a daemon, not a service. '
-                        + 'Running service names can be found with "ceph orch ls"')
-            except OrchestratorError:
-                return f'Failed to remove service. <{service_name}> was not found. Running service names can be found with "ceph orch ls"'
```

Comment:
This PR works for allowing us to remove osd services but is there a reason for removing this block? I thought the messages were useful in cases where the user tries to remove a daemon with "ceph orch rm" or maybe doesn't know the full name of the service and could use the pointer to "ceph orch ls"

Reply:
How about I could switch it so that ``ceph orch rm osd.foo`` will error out with a list of osds, and ``--force`` will proceed?